letsdrill.rb

```diff
@@ -1,24 +1,41 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+  result = ""
+  if integer > 89
+    result = "A"
+  elsif integer > 79
+    result = "B"
+  elsif integer > 69
+    result = "C"
+  elsif integer > 59
+    result = "D"
+  else
+    result = "F"
+  end
 
+  return result
 end
 
 def shortest_string(array)
+  result = array.first
 
-  #Put your code here!
+  array.each { |item|
+    if item.length < result.length
+      result = item
+    end
+  }
 
+  return result
```

Comment:
Take a look through the ruby Enumerable documentation, and see if you can't solve this with a one-liner.